wire-ios-request-strategy/Sources/Notifications/PushNotificationStatus.swift

```diff
@@ -107,7 +107,7 @@ open class PushNotificationStatus: NSObject {
         highestRankingEventId.apply(eventIdRanking.remove)
         eventIdRanking.minusSet(Set<UUID>(eventIds))
 
-        WireLogger.updateEvent.info("finished fetching all available events, last event id: \(lastEventId)")
+        WireLogger.updateEvent.info("finished fetching all available events, last event id: \(lastEventId?.uuidString ?? "<nil>")")
```

Comment:
Could we use `String(describing:)` instead, like often suggested by Xcode?

Reply:
very good point, thanks: 2ef7d855e970ea60293a8626c776f2cfc380af10